### PR TITLE
add score threshold calculator to Ray Vertex and driver

### PR DESCRIPTION
**Pull request by jakebiesinger**
@anbangx

### CODE REVIEW

**Comment by anbangx** [comment]
LGTM

**Comment by jakebiesinger** [comment]
do you have changes to add to it?

**Comment by anbangx** [comment]
I want to change the runStatsJob() to another pregel job to filter vertices. I will merge with this branch.

**Comment by jakebiesinger** [comment]
filter vertices?

**Comment by anbangx** [comment]
I think you are doing it. Never mind.

On Wed, Dec 18, 2013 at 11:45 AM, Jake Biesinger
notifications@github.comwrote:

> filter vertices?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/uci-cbcl/genomix/pull/113#issuecomment-30873857
> .

## 

Best Regards,

Anbang Xu

**Comment by jakebiesinger** [comment]
Sorry man.  Maybe you could convert it to a pregelix job?

**Comment by anbangx** [comment]
Sure. Let me know what you want me to work on.

On Wed, Dec 18, 2013 at 11:55 AM, Jake Biesinger
notifications@github.comwrote:

> Sorry man. Maybe you could convert it to a pregelix job?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/uci-cbcl/genomix/pull/113#issuecomment-30874687
> .

## 

Best Regards,

Anbang Xu

File: genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/scaffolding2/RayVertex.java [hunk 1]
```diff
@@ -613,7 +627,39 @@ public static PregelixJob getConfiguredJob(
...
+                curCounts.put(X, curCounts.get(X) + c.getCounter());
+            } else {
+                curCounts.put(X, c.getCounter());
+            }
+            total += c.getCounter();
+            count++;
+        }
+        long numSeen = 0;
+        if (topNumber == null) {
+            topNumber = (int) (count * topFraction);
```
Comment (jakebiesinger) [inline genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/scaffolding2/RayVertex.java]: boh.  this should be total \* topFraction

File: genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/scaffolding2/RayVertex.java [hunk 2]
```diff
@@ -74,8 +86,10 @@ public void compute(Iterator<RayMessage> msgIterator) throws Exception {
      * @return whether or not this node meets the "seed" criteria
      */
     private boolean isStartSeed() {
-        // TODO Auto-generated method stub
-        return getVertexId().toString().equals("CTCTTCTTACCAC");
+        float coverage = getVertexValue().getAverageCoverage(); 
+        return ((coverage >= COVERAGE_DIST_NORMAL_MEAN - COVERAGE_DIST_NORMAL_STD)
```
Comment (jakebiesinger) [inline genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/scaffolding2/RayVertex.java:90]: oops.  should be consistent with other coverage threshold :(

**Comment by jakebiesinger** [comment]
Have a peek, @anbangx.  If you approve, I'll merge it into the Ray branch, then we can merge your #111 into the branch too and resolve any conflicts.

**Comment by anbangx** [comment]
Go ahead yo merge with my PR.

**Pull request merged**